packages/tooltip/src/Tooltip/Tooltip.types.ts

```diff
@@ -23,7 +23,10 @@ export const Align = {
   Right: PopoverAlign.Right,
 } as const;
 
-export type Align = (typeof Align)[keyof typeof Align];
```

Comment:
Wasn't this already excluding `'center-vertical'` and `'center-horizontal'` on lines 19-24?

Reply:
yes it was just a different way of writing it. i can revert if you prefer

Reply:
its fine, i don't have a preference.